### ISSUE TITLE
Mike Whipp Boss Art

### ISSUE THREAD

**Issue by goofanader** [goofanader/minna-no-fighting-game#14]
### Mike Whipp
- [x] Front Facing Sprite
- [ ] Idle Animation
- [ ] Walking Animation
- [ ] Attack 1 Animation
- [ ] OPTIONAL: Death Animation
### Arcadia (Optional)
- [ ] Idle Animation
- [ ] Walking Animation
- [ ] Point to the Right Sprite
- [ ] Force Blast Sprite (hands and arms out to the sides of the screen, as if force pushing people away)
- [ ] Attack 1 Animation
- [ ] Attack 2 Animation
## References
- **References:** [Facial Hair Reference](https://drive.google.com/open?id=1RDwBCidEk8qe-MW_oeRYu-5mEL3abTfWaw), [Tochiro from Captain Harlock Reference](http://captainharlockanimeimages.tumblr.com/post/133638870832/captain-harlock-tochiro-oyama-an-eternal), [Arcadia Reference](http://www.squadron.com/v/vspfiles/photos/HE04724-3.jpg), [Arcadia Special Attack](https://theseaofstars.files.wordpress.com/2014/09/endless_odyssey_ep_04_52.jpg?w=300&h=229)
- **Opening Monologue:** I know I have something that can stop you in here somewhere!
## Additional Items/Changes to the Reference

He wants to be Tochiro from Captain Harlock, pretty much.
- **Eyes:** Glasses, possibly with a single (blue) dot for a pupil or just blue-mirrors if no pupils
- **Hair:** Ginormous hat
- **Face:** goatee + mustache see references
- **Top/Shirt:** Largely invisible because of cloak
- **Legs/Pants:** Largely invisible because of cloak
- **Feet/Shoes:** Largely invisible because of cloak
- **Body:** Largely invisible because of cloak
- **Other:** Large all-concealing brown cloak and big leather hat, possibly with bullet holes
## Colors
- **Primary:** brown
- **Secondary:** beige
- **Tertiary:** red
### Regular Attack: Unofficial Librarian

Pulls [VHS tape|DVD/CD|Game Cartridge] from cloak, throws at opponent. Does not do much damage, but has a lot of things to throw in there somehow.
### Special Attack: Arcadia Transference

When health bar runs out, character dies but his soul/consciousness/ghost becomes united with Captain Harlock's Arcadia (green version, chibi?) which again has full health and attacks the enemy.

Arcadia Reference: http://www.squadron.com/v/vspfiles/photos/HE04724-3.jpg
#### Arcadia Attacks
- **Normal Attack:** shoots the cannon on the top of the ship
- **Special Attack:** "jackknife" battering ram, see https://theseaofstars.files.wordpress.com/2014/09/endless_odyssey_ep_04_52.jpg?w=300&h=229
  
  basically a big-ass combat knife flips out of the front of the ship which then rams the opponent for massive damage.
## Phyllis Notes

We're not doing Arcadia, sorry Mike :(

**Comment by goofanader** [comment]
Added a facial hair reference picture link.

**Comment by eyfey** [comment]
![mike whipp](https://cloud.githubusercontent.com/assets/11142143/12159336/396b1d84-b496-11e5-958b-09524da0dadd.png)
HERE'S WHAT I GOT SO FAR (the cloak will probably cover more in the actual animations)

**Comment by goofanader** [comment]
OMGGGGGG IT'S SO GOOOODD!!!!!!
